tests/auth_tests/test_forms.py

```diff
@@ -867,6 +869,31 @@ def test_bug_19349_bound_password_field(self):
         # value to render correctly
         self.assertEqual(form.initial["password"], form["password"].value())
 
+    @override_settings(ROOT_URLCONF="auth_tests.urls_admin")
+    def test_bug_34066_link_to_password_reset_in_helptext(self):
```

Comment:
Lastly we should mention what the specific issue is about the link in the helptext 🙂
```suggestion
    def test_bug_34066_link_to_password_reset_in_helptext(self):
        # The password reset link should always refer to the primary key even when accessed via a to_field
```

Reply:
Please chop the ticket number from the test method name.